ehrapy/preprocessing/_bias.py

```diff
@@ -0,0 +1,244 @@
+import itertools
+from collections.abc import Iterable
+from typing import Literal
+
+import numpy as np
+import pandas as pd
+from anndata import AnnData
+
+from ehrapy.anndata import anndata_to_df, check_feature_types
+from ehrapy.anndata._constants import CATEGORICAL_TAG, CONTINUOUS_TAG, DATE_TAG, FEATURE_TYPE_KEY
+
+
+@check_feature_types
+def detect_bias(
+    adata: AnnData,
+    sensitive_features: Iterable[str] | Literal["all"],
+    *,
+    run_feature_importances: bool | None = None,
+    corr_threshold: float = 0.5,
+    smd_threshold: float = 0.5,
+    categorical_factor_threshold: float = 2,
+    feature_importance_threshold: float = 0.1,
+    prediction_confidence_threshold: float = 0.5,
+    corr_method: Literal["pearson", "spearman"] = "spearman",
+    copy: bool = False,
+) -> dict[str, pd.DataFrame] | tuple[dict[str, pd.DataFrame], AnnData]:
+    """Detects biases in the data using feature correlations, standardized mean differences, and feature importances.
+
+    Detects biases with respect to sensitive features, which can be either a specified subset of features or all features in adata.var.
+    The method detects biases by computing:
+    - pairwise correlations between features
```

Comment:
The bullet points don't seem to render properly:

https://ehrapy--690.org.readthedocs.build/en/690/usage/preprocessing/ehrapy.preprocessing.detect_bias.html

You might need more line breaks or something

Reply:
Thanks for spotting that - it's fixed now.